### ISSUE TITLE
Fix organisation list accessibility

### ISSUE THREAD

**Issue by SiniCode** [UniversityOfHelsinkiCS/palaute#1661]
- [ ] Add aria-labels to links that tell where the link takes
- [ ] Investigate why screen reader announces Active surveys page after clicking the link even though it should announce the organisation name
